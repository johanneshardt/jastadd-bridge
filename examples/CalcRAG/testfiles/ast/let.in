let area = 2.0 * 3.0 in 4.0 * area end