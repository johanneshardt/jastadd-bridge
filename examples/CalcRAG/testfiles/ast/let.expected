Program
  Let
    List
      Binding
        IdDecl ID="area"
        Mul
          Numeral NUMERAL="2.0"
          Numeral NUMERAL="3.0"
    Mul
      Numeral NUMERAL="4.0"
      IdUse ID="area"
